tests/test_ticker.py

```diff
@@ -323,6 +325,15 @@ def test_earnings_dates_with_limit(self):
     #     data_cached = self.ticker.earnings_trend
     #     self.assertIs(data, data_cached, "data not cached")
 
+    def test_ticker_has_tz(self):
+        test_data = {"O2D.DE": "UTC", "LHA.DE": "Europe/Berlin"}
```

Comment:
I can't see O2D.DE on Yahoo - is it real?

Reply:
It may be delisted now, but returned earning dates with UTC timestamp. Let me check this and find another test related symbol.